project/Dependencies.scala

```diff
@@ -689,17 +688,8 @@ object Dependencies {
     akkaPersistenceCassandra
   )
 
-  val `kafka-server` = libraryDependencies ++=
-    // log4j version prior to 1.2.17 required javax.jms, and that artifact could not properly resolved when using maven
-    // without adding a resolver. The problem doesn't appear with sbt because the log4j version brought by both zookeeper
-    // and curator dependencies are evicted to version 1.2.17. Unfortunately, because of how maven resolution works, we
-    // have to explicitly add the desired log4j version we want to use here.
-    // By the way, log4j 1.2.17 and later resolve the javax.jms dependency issue by using geronimo-jms. See
-    // http://stackoverflow.com/questions/4908651/the-following-artifacts-could-not-be-resolved-javax-jmsjmsjar1-1
-    // for more context.
```

Comment:
I'm sure this comment was true at the time it was written, but I'm not seeing a transitive dependency to any JMS implementation now. Presumably ZooKeeper and Curator are excluding it.

Reply:
I found this: https://issues.apache.org/jira/browse/ZOOKEEPER-721 apparently fixed in 2012! 🤷‍♂️